packages/redux/src/selectors.ts

```diff
@@ -34,80 +34,82 @@ export const getFileStorage = <State extends RootState = RootState>(
 
 export const getWorkspaceStorage = <State extends RootState = RootState>(
   store: State
-): Storage['workspaceStorage'] => getStorage(store).workspaceStorage;
+): State['storage']['workspaceStorage'] => getStorage(store).workspaceStorage;
 
 // Workspace
 export const getWorkspace = <State extends RootState = RootState>(
   store: State
-): RootState['workspace'] => store.workspace;
+): State['workspace'] => store.workspace;
 
 // Settings
+type Settings<
+  State extends RootState = RootState
+> = State['workspace']['data']['settings'];
+
```

Comment:
Minor, but I would move this above the selectors probably. Just where I tend to put utils/types so they don't blend into implementation

Also, probably doesn't need a default generic value. In this case, it could make it easier to forget and not make the return type generic. `Settings['defaultDateTimeFormat']` would be valid, but always refer to the `RootState` type